docset/windows/appx/Set-NonRemovableAppsPolicy.md

```diff
@@ -0,0 +1,201 @@
+
+## EXAMPLES
+
+### Example 1: Set the app package Application1 as non-removable
+```
+PS C:\> Set-NonRemovableAppsPolicy -Online -PackageFamilyName Application1_1.0.0.0+x64__ms7gsqeatfeb6 -NonRemovable 1
+```
+
+This command gets information about all installed app packages which have been previously configured as non-removable.
+
+### Example 2: Set the app package Application1 as removable
+```
+PS C:\> Set-NonRemovableAppsPolicy -Online -PackageFamilyName Application1_1.0.0.0+x64__ms7gsqeatfeb6 -NonRemovable 0
+```
+
+### Example 3: Sets the app package Application1 as non-removable on an offline windows image
+```
+PS C:\> Get-NonRemovableAppsPolicy -Path ".\wim\image.wim"
+```
+
+This command gets all apps packages that have been loaded into the offline operating system image.
+
+## PARAMETERS
+
+### -PackageFamilyName
+Specifies the Package Family Name of the app package that will have it's non-removable status set.
+
+|||
+|-|-|
+| Type: | String |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -NonRemovable
+Specifies that the app package will be configured as non-removable or not. Accepted values are as follows:
+- 1 = Sets the application as non-removable, and unable to be uninstalled.
+- 0 = Sets the application as removable, and capable of being uninstalled.
+
+|||
+|-|-|
+| Type: | Integer |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -Online
+Indicates that the cmdlet operates on a running operating system on the local host.
+
+|||
+|-|-|
+| Type: | SwitchParamter |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -Path
+Specifies the full path to the root directory of the offline Windows image that you will service. If the directory named Windows is not a subdirectory of the root directory, WindowsDirectory must be specified.
+
+|||
+|-|-|
+| Type: | String |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -ScratchDirectory
+Specifies a temporary directory that will be used when extracting files for use during servicing. The directory must exist locally. If not spcified, the `\Windows\%Temp%` directory will be used, with a subdirectoryname of a randomly generated hexadecimal value for each run of DISM. Items in the scratch directory are deleted after each operation. You should not use a network share location as a scratch directory to expand a pachage(.cab or .msu file) for installation. The directory used for extracting files for temporary usage during servicing should be a local directory.
```

Comment:
Fix the spelling for 'specified' and remove the % from 'Windows\Temp' as that is a static path. 'Subdirectoryname' needs a space been directory and name. The last sentence can be removed.

Reply:
Updated.